README.md

```diff
@@ -5,8 +5,8 @@ Provide reusable typescript and react eslint configs
 1. Install the peerDependencies of eslint configs:
```

Comment:
nit: we basically duplicate readmes now, should we just link to the individual readmes from here?

Reply:
Make sense